src/fields.jl

```diff
@@ -52,3 +52,231 @@ Field(LX, data, grid::Grid1D) = Field1D(LX, data, grid)
 Constructs a 2D field of `data` at location `(LX, LY)` on `grid`.
 """
 Field(LX, LY, data, grid::Grid2D) = Field2D(LX, LY, data, grid)
+
+############################################################################
+#-------------------------------Interpolation------------------------------#
```

Comment:
What happened here? Didn't we change this section breaking? Seems that old style has infiltrated back?

Reply:
(Sorry if I'm a bit OCD about this things...)

Reply:
No problems. I'm sure all of this helps in readability.

Reply:
But what happened? I remember you committed my changes but then you changed your mind?